src/rpc.rs

```diff
@@ -44,7 +44,7 @@ impl RpcClient for UPClientZenoh {
 
         // Create UAttributes and put into Zenoh user attachment
         let uattributes = UAttributes::request(
-            UUIDBuilder::new().build(),
+            UUIDBuilder::build(),
             topic,
             self.get_response_uuri(),
             options.clone(),
```

Comment:
Sorry, this part is not in the section I can leave a review comment on, but again something I noticed and wanted to check on.

Same kind of question as in [this comment](https://github.com/eclipse-uprotocol/up-client-zenoh-rust/pull/24/files?diff=split&w=0#r1563253578)

Reply:
Same as [here](https://github.com/eclipse-uprotocol/up-client-zenoh-rust/pull/24#discussion_r1565149160). `req_id` is only used in the uProtocol layer and Zenoh doesn't need it in the transmission.